src/api/app/models/staging/staging_project.rb

```diff
@@ -16,7 +16,7 @@ def untracked_requests
     # The requests to review are requests which are not related to the staging project (unless they are also staged).
     # They simply need a review from the maintainers of the staging project.
     def requests_to_review
-      @requests_to_review ||= BsRequest.with_open_reviews_for(by_project: name)
+      @requests_to_review ||= BsRequest.with_open_reviews_for(by_project: name).includes(:reviews)
```

Comment:
wouldn't it make sense to add this `includes` inside `with_open_reviews_for`? :thinking: